chapter_computer-vision/kaggle-cifar10.md

```diff
@@ -513,8 +513,8 @@ train(net, train_iter, valid_iter, num_epochs, lr, wd, devices, lr_period,
 
 ```{.python .input}
 #@tab pytorch
-devices, num_epochs, lr, wd = d2l.try_all_gpus(), 5, 0.1, 5e-4
-lr_period, lr_decay, net = 50, 0.1, get_net()
```

Comment:
What did you observe when lr=0.02?

Reply:
With `lr=0.02`, train loss decreased a bit at the start and then was constant. While `train acc` and `valid acc` remained almost zero.